common/types.go

```diff
@@ -116,56 +116,6 @@ func (l *LogLevel) Parse(s string) error {
 	return err
 }
 
-type FileType int
```

Comment:
Was there a purpose for these types? They arent used anywhere

Reply:
Might be during jacobsa days were used to detect the file-type, with fuse might not be of much use.

Reply:
With libfuse these checks for filetypes are done as part of method "fillStat" which is relying on attribute structure.

Reply:
ok, so I'm good to remove them then